clion/src/191/main/kotlin/org/rust/clion/profiler/RsSymbolSearcher.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.clion.profiler
+
+import com.intellij.openapi.application.ApplicationManager
+import com.intellij.openapi.project.Project
+import com.intellij.profiler.model.NativeCall
+import com.intellij.psi.search.GlobalSearchScope
+import org.rust.lang.core.psi.RsFunction
+import org.rust.lang.core.psi.ext.qualifiedName
+import org.rust.lang.core.stubs.index.RsNamedElementIndex
+import org.rust.openapiext.getElements
+
+class RsSymbolSearcher(private val project: Project) {
+    fun find(nativeCall: NativeCall): Array<RsFunction> {
```

Comment:
Let's add some tests for this method with some real data in `NativeCall`
Now it's rather hard to understand is it correct or not (and harder to maintain it in a correct state)